src/lib/checkbox/checkbox.html

```diff
@@ -18,8 +18,7 @@
     <div md-ripple *ngIf="!_isRippleDisabled()" class="md-checkbox-ripple"
          [mdRippleTrigger]="_getHostElement()"
          [mdRippleCentered]="true"
-         [mdRippleSpeedFactor]="0.3"
-         mdRippleBackgroundColor="rgba(0, 0, 0, 0)"></div>
+         [mdRippleSpeedFactor]="0.3"></div>
```

Comment:
As discussed on Slack, we are going to have another PR to make all component ripples align with the specs.